src/audio/src/src_ipc.h

```diff
@@ -0,0 +1,19 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2017-2024 Intel Corporation
+ */
+
+#ifndef __SOF_AUDIO_SRC_SRC_IPC_H__
+#define __SOF_AUDIO_SRC_SRC_IPC_H__
+
+#include <sof/audio/ipc-config.h>
+#include <ipc4/base-config.h>
+#include <stdint.h>
+
+/* src component private data */
+struct ipc4_config_src {
+	struct ipc4_base_module_cfg base;
```

Comment:
Should we have size at the start ?

Reply:
ok, but this is an IPC header, so this should be for host/module configuration.

Reply:
It's not clear to me what you are suggesting to do. Should these IPC headers be somewhere else or use some more consistent directory locations and naming?

Reply:
@lgirdwood Ping, can you help me fix this. I don't think this PR has any other opens left.

Reply:
ok, we can address this later as I see this structure is used as header for module data in other modules.